tools/log-viewer/scripts.js

```diff
@@ -143,6 +143,23 @@ function writeLoginMessage(owner, repo) {
   return 'You need to sign in to this project\'s sidekick view the requested logs.';
 }
 
+function registerAdminDetailsListener(buttons) {
+  buttons.forEach((button) => {
+    button.addEventListener('click', async () => {
+      const url = new URL(button.dataset.url);
+      const { default: getModal } = await import('../../scripts/modal/modal.js');
+      if (url) {
+        const res = await fetch(url);
+        const jsonContent = await res.json();
+        const simpleModal = await getModal(url.pathname, () => `<code>
+            <pre>${JSON.stringify(jsonContent, null, 3)}</pre>
+          </code>`);
+        simpleModal.showModal();
```

Comment:
nice to have, but not must. can we close the modal if the user clicks outside of it?

https://github.com/adobe/aem-block-collection/blob/main/blocks/modal/modal.js#L34

Reply:
good point. I just switched it to the new modal block in boilerplate so it has all the goodies.